Tests assert particular order of XML tags which is not returned by unparse

This happens roughly every other test run - for me only on Python 2.6.

```
======================================================================
FAIL: test_multiple_roots_nofulldoc (test_dicttoxml.DictToXMLTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/piotr/TyMaszWeb/xmltodict/tests/test_dicttoxml.py", line 65, in test_multiple_roots_nofulldoc
    self.assertEqual(xml, '<a>1</a><b>2</b>')
AssertionError: u'<b>2</b><a>1</a>' != '<a>1</a><b>2</b>'

________________________________________________ summary ________________________________________________
  py27: commands succeeded
  py34: commands succeeded
  pypy: commands succeeded
ERROR:   py26: commands failed

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -29,6 +29,9 @@ before_install:
   - export JYTHON_URL='http://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.7-b3/jython-installer-2.7-b3.jar'
   - if [ "$JYTHON" == "true" ]; then wget $JYTHON_URL -O jython_installer.jar; java -jar jython_installer.jar -s -d $HOME/jython; export PATH=$HOME/jython:$PATH; jython ez_setup.py; $HOME/jython/bin/easy_install nose; fi
 
+install:
+  - if [ "$TRAVIS_PYTHON_VERSION" == "2.6" ]; then pip install ordereddict ; fi
+
 before_script: if [ "$JYTHON" == "true" ]; then export NOSE=$HOME/jython/bin/nosetests NOSE_OPTIONS=""; else export NOSE=nosetests NOSE_OPTIONS="--with-coverage --cover-package=xmltodict"; fi
 
 script: $NOSE $NOSE_OPTIONS

--- a/tests/test_dicttoxml.py
+++ b/tests/test_dicttoxml.py
@@ -80,19 +80,18 @@ class DictToXMLTestCase(unittest.TestCase):
         self.assertEqual(_strip(unparse(parse(xml))),
                          '<a><d></d>abcefg</a>')
 
-    if hasattr(collections, 'OrderedDict'):
-        def test_preprocessor(self):
-            obj = {'a': OrderedDict((('b:int', [1, 2]), ('b', 'c')))}
+    def test_preprocessor(self):
+        obj = {'a': OrderedDict((('b:int', [1, 2]), ('b', 'c')))}
 
-            def p(key, value):
-                try:
-                    key, _ = key.split(':')
-                except ValueError:
-                    pass
-                return key, value
+        def p(key, value):
+            try:
+                key, _ = key.split(':')
+            except ValueError:
+                pass
+            return key, value
 
-            self.assertEqual(_strip(unparse(obj, preprocessor=p)),
-                             '<a><b>1</b><b>2</b><b>c</b></a>')
+        self.assertEqual(_strip(unparse(obj, preprocessor=p)),
+                         '<a><b>1</b><b>2</b><b>c</b></a>')
 
     def test_preprocessor_skipkey(self):
         obj = {'a': {'b': 1, 'c': 2}}
@@ -105,32 +104,31 @@ class DictToXMLTestCase(unittest.TestCase):
         self.assertEqual(_strip(unparse(obj, preprocessor=p)),
                          '<a><c>2</c></a>')
 
-    if hasattr(collections, 'OrderedDict') and not IS_JYTHON:
+    if not IS_JYTHON:
         # Jython's SAX does not preserve attribute order
         def test_attr_order_roundtrip(self):
             xml = '<root a="1" b="2" c="3"></root>'
             self.assertEqual(xml, _strip(unparse(parse(xml))))
 
-    if hasattr(collections, 'OrderedDict'):
-        def test_pretty_print(self):
-            obj = {'a': OrderedDict((
-                ('b', [{'c': [1, 2]}, 3]),
-                ('x', 'y'),
-            ))}
-            newl = '\n'
-            indent = '....'
-            xml = dedent('''\
-            <?xml version="1.0" encoding="utf-8"?>
-            <a>
-            ....<b>
-            ........<c>1</c>
-            ........<c>2</c>
-            ....</b>
-            ....<b>3</b>
-            ....<x>y</x>
-            </a>''')
-            self.assertEqual(xml, unparse(obj, pretty=True,
-                                          newl=newl, indent=indent))
+    def test_pretty_print(self):
+        obj = {'a': OrderedDict((
+            ('b', [{'c': [1, 2]}, 3]),
+            ('x', 'y'),
+        ))}
+        newl = '\n'
+        indent = '....'
+        xml = dedent('''\
+        <?xml version="1.0" encoding="utf-8"?>
+        <a>
+        ....<b>
+        ........<c>1</c>
+        ........<c>2</c>
+        ....</b>
+        ....<b>3</b>
+        ....<x>y</x>
+        </a>''')
+        self.assertEqual(xml, unparse(obj, pretty=True,
+                                      newl=newl, indent=indent))
 
     def test_encoding(self):
         try:

--- a/tox.ini
+++ b/tox.ini
@@ -2,6 +2,7 @@
 envlist = py27,py34,pypy,py26
 [testenv]
 deps =
-  nose
   coverage
+  nose
+  ordereddict
 commands=nosetests --with-coverage --cover-package=xmltodict

